app/services/course/material/download_service.rb

```diff
@@ -0,0 +1,64 @@
+class Course::Material::DownloadService
```

Comment:
I want ZipDownloadService because it might be downloading one file alone.

I'm not inclined to support other formats.